client/asset/btc/rpcclient.go

```diff
@@ -205,7 +202,13 @@ func (wc *rpcClient) callHashGetter(method string, args anylist) (*chainhash.Has
 // getBlock fetches the MsgBlock.
 func (wc *rpcClient) getBlock(h chainhash.Hash) (*wire.MsgBlock, error) {
 	var txB dex.Bytes
-	err := wc.call(methodGetBlock, anylist{h.String(), 0}, &txB)
+	args := anylist{h.String()}
+	if wc.booleanGetBlock {
+		args = append(args, false)
+	} else {
+		args = append(args, 0)
```

Comment:
I just caught this discrepancy server-side too, where we were passing `true` instead of `1` to get the `GetBlockVerboseResult`.  We were/are basically getting lucky there since bitcoind/litecoind were happy to treat `true` as `1` even though they really want 0,1, or 2.

I'm planning to make the necessary changes in https://github.com/decred/dcrdex/pull/1581 since I'm also fixing up the LiveUTXO btc tests and I wanted the mgsBlock... and I noticed this issue.